src/pages/new-claim/new-claim-form.tsx

```diff
@@ -0,0 +1,114 @@
+import { FormEventHandler } from 'react';
+import { goSync } from '@api3/promise-utils';
+import { BigNumber, utils } from 'ethers';
+import isEmpty from 'lodash/isEmpty';
+import Input from '../../components/input';
+import Button from '../../components/button';
+import { Policy } from '../../chain-data';
+import globalStyles from '../../styles/global-styles.module.scss';
+import styles from './new-claim-form.module.scss';
```

Comment:
Locally this file was renamed from the previous `new-claim.tsx` file, but GH has decided to present it as a new file instead. The only logic that has changed from before is the `handleSubmit` function